tests/security/grub_auth/grub_authorization.pm

```diff
@@ -31,7 +31,8 @@ my $test_passwd = 'pw_test';
 sub switch_boot_menu {
     my $switch = shift;
 
-    if (get_var('UEFI')) {
+    # UEFI item is available only from 15-SP4
+    if (get_var('UEFI') && (!is_sle('<15-SP4'))) {
```

Comment:
Is it equal `if (get_var('UEFI') && is_sle('>=15-SP4')) {` ?

Reply:
I don't think so, because `is_sle('>=15-SP4')` will evaluate to `FALSE` with Tumbleweed.